lib/plugins/usermanager/cli.php

```diff
@@ -0,0 +1,279 @@
+        $options->registerArgument('group', 'Group(s), comma-seperated', true, 'addtogroup');
+
+        // remove from group
+        $options->registerCommand('removefromgroup', 'Remove user from group(s)');
+        $options->registerArgument('name', 'Username', true, 'removefromgroup');
+        $options->registerArgument('group', 'Group(s), comma-seperated', true, 'removefromgroup');
+    }
+
+    /** @inheritdoc */
+    protected function main(\splitbrain\phpcli\Options $options)
+    {
+        switch ($options->getCmd()) {
+            case 'list':
+                $ret = $this->cmdList($options->getOpt('verbose'));
+                break;
+            case 'add':
+                $ret = $this->cmdAdd($options->getOpt('notify'), $options->getArgs());
+                break;
+            case 'delete':
+                $ret = $this->cmdDelete($options->getArgs());
+                break;
+            case 'addtogroup':
+                $ret = $this->cmdAddToGroup($options->getArgs());
+                break;
+            case 'removefromgroup':
+                $ret = $this->cmdRemoveFromGroup($options->getArgs());
+                break;
+
+            default:
+                echo $options->help();
+                $ret = 0;
+        }
+
+        exit($ret);
+    }
+
+    /**
+     * @param bool $showdetails
+     * @return int
+     */
+    protected function cmdList(bool $showdetails)
+    {
+        global $auth;
+
+        if (!isset($auth)) {
+            $this->error($this->getLang('noauth'));
+            return 1;
+        } elseif (!$auth->canDo('getUsers')) {
+            $this->error($this->getLang('nosupport'));
+            return 1;
+        } else {
+            $this->listUsers($showdetails);
+        }
+
+        return 0;
+    }
+
+    /**
+     * List the given users
+     *
+     * @param bool $details display details
+     */
+    protected function listUsers(bool $details = False)
+    {
+        global $auth;
+        $list = $auth->retrieveUsers();
+
+        $tr = new TableFormatter($this->colors);
+
+        foreach ($list as $username => $user) {
+            $content = [$username];
+            if ($details) {
+                array_push($content, $user['name']);
+                array_push($content, $user['mail']);
+                array_push($content, implode(", ", $user['grps']));
+            }
+            echo $tr->format(
+                [15, 25, 25, 15],
+                $content
+            );
+        }
+    }
+
+    /**
+     * Adds an user
+     *
+     * @param bool $notify display details
+     * @param array $args
+     * @return int
+     */
+    protected function cmdAdd(bool $notify, array $args)
+    {
+        global $auth;
+
+        if (!$auth->canDo('addUser')) {
+            $this->error($this->getLang('nosupport'));
+            return 1;
+        }
+
+        list($user, $name, $mail, $grps, $pass) = $args;
```

Comment:
The help text says, the order is `add <OPTIONS> <name> <mail> [<full_name>] [<groups>] [<password>]`. And indeed creating a user now puts the email address into the name field.